boot/freeldr/freeldr/lib/fs/fs.c

```diff
@@ -51,29 +51,7 @@ typedef struct tagDEVICE
 static FILEDATA FileData[MAX_FDS];
 static LIST_ENTRY DeviceListHead;
 
-#define IS_VALID_FILEID(FileId) \
-    ((ULONG)(FileId) < _countof(FileData) && FileData[(ULONG)(FileId)].FuncTable)
-
-typedef const DEVVTBL* (*PFS_MOUNT)(ULONG DeviceId);
-
-PFS_MOUNT FileSystems[] =
-{
-#ifndef _M_ARM
-    IsoMount,
-#endif
-    FatMount,
-    BtrFsMount,
-#ifndef _M_ARM
-    NtfsMount,
-    Ext2Mount,
-#endif
-#if defined(_M_IX86) || defined(_M_AMD64)
-#ifndef UEFIBOOT
-    PxeMount,
-#endif
-#endif
-};
-
```

Comment:
what has that stripped block to do with your PR? Is that a merge-issue maybe?

Reply:
Yes rebase-issue, I'm going to fix this immediately.

Reply:
and i'm not very good in Git

Reply:
Now it's fixed

Reply:
confirmed and discussion resolved. I will leave the judgement whether this is a good PR or just a workaround to other people though.